specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -3506,6 +3511,22 @@
           "maxLength": 16384
         }
       }
+    },
+    "MetaDataContract": {
+      "type": "object",
+      "description": "A combination of user defined name and user specified data and recognition model name for largePersonGroup/personGroup, and largeFaceList/faceList.",
+      "properties": {
+        "RecognitionModel": {
```

Comment:
camelCasing (RecognitionModel -> recognitionModel). Please double check other places.

Reply:
No need to change here.